projects/angular/src/forms/datepicker/_variables.datepicker.scss

```diff
@@ -3,41 +3,55 @@
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
-@use '../../utils/variables/variables';
+@use '@cds/core/tokens/tokens.scss';
+
+$clr-calendar-click-target: #{tokens.$cds-global-space-11};
```

Comment:
Why do we hardcode some values here, while others need a property? 
Wasn't the idea to use this file as compile time checking only? Now it still behaves like a secondary theme layer (SCSS based).
If we don't need a property and want to hardcode something, we can easily do it in the component scss file itself.

Reply:
In this example `$clr-calendar-click-target` don't have property to be associated with (`--clr-calendar-click-target`), but is old (before TM) variable that might be used from the client applications.

The idea is to keep it and replace it's `CLR value` with `CDS token`.

Reply:
I think we can bring this into the component file.
Click target areas should be system wide usability concerns. If it does not meet a11y criteria, we'd better fix it than expose it to the user. It's a too deep technical detail for me.
If you prefer it to stay this way, I am not strongly against it. You can dismiss this comment and merge it.